packages/profile/src/components/trophies/leaderboard.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  cn,
+  SpaceInvaderIcon,
+  SparklesIcon,
+  StateIconProps,
+} from "@cartridge/ui-next";
+import { Link } from "react-router-dom";
+import { Player } from ".";
+
+export function Leaderboard({
+  players,
+  address,
+}: {
+  players: Player[];
+  address: string;
+}) {
+  return (
+    <div className="flex flex-col gap-y-px rounded-md overflow-hidden">
+      {players
+        .sort((a, b) => a.rank - b.rank)
```

Comment:
nit: I assume the sort exists for mocked data? graphql response should be already sorted or has sort param.